drivers/bluetooth/hci/Kconfig

```diff
@@ -85,12 +85,20 @@ config BT_SPI_DEV_NAME
 
 if BT_SPI
 
+config BT_BLUENRG_ACI
+	bool "Enable ACI message with with BlueNRG-based devices"
+	default n
+	help
+	  Enable support for devices compatible with the BlueNRG Bluetooth
+	  Stack. Current driver supports: ST BLUENRG-MS.
+
+
```

Comment:
Redundant empty line here

Reply:
Looks like there's one in the commit message as well (which github doesn't let me do inline commenting on). If these minor issues are fixed I'd be happy to approve this PR.

Reply:
@jhedberg Modification done, Thanks for the review